src/App.jsx

```diff
@@ -0,0 +1,31 @@
+import './App.css'
+import { BrowserRouter, Route, Routes } from 'react-router-dom'
+import HomePage from 'pages/HomePage'
+import PlantPage from 'pages/Plant/PlantPage'
+import { Box } from '@mui/material'
+import GoogleFontLoader from 'react-google-font-loader';
+
+function App() {
+  return (
+    <Box component="section" sx={{ display: "flex", justifyContent: "center", width: "100%" }}>
+      <GoogleFontLoader
+        fonts={[
+          {
+            font: 'Roboto',
+            weights: [200, 400, 500, 700],
+          },
+        ]}
+      />
+      <BrowserRouter>
+        <Routes>
+          <Route path="/" element={<HomePage />} exact />
```

Comment:
`react-router-dom` v6 doesn't need to use an exact prop anymore


You can read it here: [New changes in the react-router-dom v6](https://stackoverflow.com/a/49162423)
In the last line of his comment, he mentioned the following content:
> You don't need to use an exact prop on anymore. This is because all paths match exactly by default. If you want to match more of the URL because you have child routes use a trailing * as in